components/kyma-environment-broker/CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,3 @@
 # Code of conduct
 
-Each contributor and maintainer of this project agrees to follow the [community Code of Conduct](https://github.com/kyma-project/community/blob/main/contributing/01-code-of-conduct.md) that relies on the CNCF Code of Conduct. Read it to learn about the agreed standards of behavior, shared values that govern our community, and details on how to report any suspected Code of Conduct violations.
+Each contributor and maintainer of this project agrees to follow the [community Code of Conduct](https://github.com/kyma-project/community/blob/main/docs/contributing/01-code-of-conduct.md) that relies on the CNCF Code of Conduct. Read it to learn about the agreed standards of behavior, shared values that govern our community, and details on how to report any suspected Code of Conduct violations.
```

Comment:
```suggestion
Each contributor and maintainer of this project agrees to follow the [community Code of Conduct](https://github.com/kyma-project/community/blob/main/docs/contributing/01-code-of-conduct.md) that relies on the CNCF Code of Conduct. Please read it to learn about the agreed standards of behaviour, shared values that govern our community, and details on how to report any suspected Code of Conduct violations.
```

Reply:
Are we doing British or American English at Kyma?

Reply:
We use American English and for the sake of the simplicity we avoid using any unessential words, including "please". If you'd actually like to have a look at the style and terminology guidelines, here is the link: https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md